docs/en/user_guides/1_config.md

```diff
@@ -360,15 +483,105 @@ model = dict(
+        'Note that the quotation marks are necessary and that no white space '
+        'is allowed.')
+    args = parser.parse_args()
+    return args
+
+def main():
+    args = parse_args()
+
+    cfg = Config.fromfile(args.config)
+    if args.cfg_options is not None:
+        cfg.merge_from_dict(args.cfg_options)
+
+    print(cfg)
+
+if __name__ == '__main__':
+    main()
+```
+
+A example config file `demo_config.py` as follows:
+
+```python
+backbone = dict(
+    type='ResNetV1c',
+    depth=50,
+    num_stages=4,
+    out_indices=(0, 1, 2, 3),
+    dilations=(1, 1, 2, 4),
+    strides=(1, 2, 1, 1),
+    norm_eval=False,
+    style='pytorch',
+    contract_dilation=True)
+```
+
+Run `demo_script.py`:
+
+```shell
+python demo_script.py demo_config.py
+```
+
+```shell
+Config (path: demo_config.py): {'backbone': {'type': 'ResNetV1c', 'depth': 50, 'num_stages': 4, 'out_indices': (0, 1, 2, 3), 'dilations': (1, 1, 2, 4), 'strides': (1, 2, 1, 1), 'norm_eval': False, 'style': 'pytorch', 'contract_dilation': True}}
+```
+
+Modify config through script arguments:
+
+```shell
+python demo_script.py demo_config.py --cfg-options backbone.depth=101
+```
+
+```shell
+Config (path: demo_config.py): {'backbone': {'type': 'ResNetV1c', 'depth': 101, 'num_stages': 4, 'out_indices': (0, 1, 2, 3), 'dilations': (1, 1, 2, 4), 'strides': (1, 2, 1, 1), 'norm_eval': False, 'style': 'pytorch', 'contract_dilation': True}}
+```
+
 - Update values of list/tuples.
 
-  If the value to be updated is a list or a tuple. For example, the config file normally sets `sigma_range=(0, 0.04)` in `data_preprocessor` of `model`.
+  If the value to be updated is a list or a tuple. For example, the config file `demo_config.py` sets `strides=(1, 2, 1, 1)` in `backbone`.
   If you want to change this key, you may specify in two ways:
 
-  1. `--cfg-options model.data_preprocessor.sigma_range="(0, 0.05)"`. Note that the quotation mark " is necessary to support list/tuple data types.
-  2. `--cfg-options model.data_preprocessor.sigma_range=0,0.05`. Note that **NO** white space is allowed in the specified value.
+  1. `--cfg-options backbone.strides="(1, 1, 1, 1)"`. Note that the quotation mark " is necessary to support list/tuple data types.
+
+     ```shell
+     python demo_script.py demo_config.py --cfg-options backbone.strides="(1, 1, 1, 1)"
+     ```
+
+     ```shell
+     Config (path: demo_config.py): {'backbone': {'type': 'ResNetV1c', 'depth': 50, 'num_stages': 4, 'out_indices': (0, 1, 2, 3), 'dilations': (1, 1, 2, 4), 'strides': (1, 1, 1, 1), 'norm_eval': False, 'style': 'pytorch', 'contract_dilation': True}}
+     ```
+
+  2. `--cfg-options backbone.strides=1,1,1,1`. Note that **NO** white space is allowed in the specified value.
      In addition, if the original type is tuple, it will be automatically converted to list after this way.
 
+     ```shell
+     python demo_script.py demo_config.py --cfg-options backbone.strides="(1, 1, 1, 1)"
```

Comment:
I think there should be` --cfg-options backbone.strides=1, 1, 1, 1`

Reply:
> I think there should be` --cfg-options backbone.strides=1, 1, 1, 1`

That will report an error.
https://github.com/open-mmlab/mmsegmentation/blob/6cdc2c4a8a4f3abc71ee8acdd52c0a46326abe4c/docs/en/user_guides/1_config.md?plain=1#L369